iBurn/MoreViewController.swift

```diff
@@ -100,6 +103,23 @@ class MoreViewController: UITableViewController, SKStoreProductViewControllerDel
         }
     }
     
+    @IBAction func updatesToggled(sender: UISwitch) {
```

Comment:
I'm thinking of making a whole new screen that has more detail about the updates as well, like when each data type was last updated, when we last checked, etc.